src/sass/_shame.scss

```diff
@@ -31,6 +31,11 @@ body .row {
   border: 1px solid $color-gray-lightest;
 }
 
+.schemaform-intro {
```

Comment:
I think you can put this in edu-benefits.scss for now. In the near future we'll need to pull all the schemaform specific stuff into it's own file, but it's all in edu-benefits for the moment.